packages/playwright/src/matchers/toMatchText.ts

```diff
@@ -49,17 +49,17 @@ export async function toMatchText(
     !(expected && typeof expected.test === 'function')
   ) {
     throw new Error(
-        this.utils.matcherErrorMessage(
+        matcherErrorMessage(
```

Comment:
Looking at the `matcherErrorMessage` implementation, perhaps just inline it? We can also inline COLORs in a separate PR.